frontend/src/api/k8s/servingRuntimes.ts

```diff
@@ -242,6 +277,7 @@ export const updateServingRuntime = (options: {
   initialAcceleratorProfile?: AcceleratorProfileState;
   selectedAcceleratorProfile?: AcceleratorProfileSelectFieldState;
   isModelMesh?: boolean;
+  nimPVCName?: string;
```

Comment:
This was unnecessary -- you don't support edit, updating the serving runtime is not done currently.